metasim/sim/isaacgym/isaacgym.py

```diff
@@ -532,6 +556,14 @@ def simulate(self) -> None:
 
         # self.gym.sync_frame_time(self.sim)
 
+    def compute_torques(self):
+        self.actions = self.actions.view(self._num_envs, self._num_dof)
+        dof_pos = self._dof_states.view(self.num_envs, self._num_dof, 2)[..., 0]
+        dof_vel = self._dof_states.view(self.num_envs, self._num_dof, 2)[..., 1]
+        _torques = self.p_gains * (self.actions - (dof_pos - self.default_dof_pos)) - self.d_gains * dof_vel
+        self.torques = torch.clip(_torques, -self.torque_limits, self.torque_limits)
+        self.gym.set_dof_actuation_force_tensor(self.sim, gymtorch.unwrap_tensor(self.torques.to(torch.float32)))
```

Comment:
Since the method is called `compute_torques`, it would be better to return a torque. Also the method would be better to be private.
```suggestion
    def _compute_torques(self, actions: torch.Tensor):
        actions = actions.view(self._num_envs, self._num_dof)
        dof_pos = self._dof_states.view(self.num_envs, self._num_dof, 2)[..., 0]
        dof_vel = self._dof_states.view(self.num_envs, self._num_dof, 2)[..., 1]
        _torques = self.p_gains * (self.actions - (dof_pos - self.default_dof_pos)) - self.d_gains * dof_vel
        torques = torch.clip(_torques, -self.torque_limits, self.torque_limits)
        return torques
```

Reply:
we need the save the torque for torque related reward compute, save torque somewhere is necessary. I will modfied the method as private.